trestle/core/catalog/catalog_writer.py

```diff
@@ -190,14 +192,21 @@ def _construct_set_parameters_dict(
             # adds display name, if no display name then do not add to dict
             if display_name != '' and display_name is not None:
                 new_dict[const.DISPLAY_NAME] = display_name
+            # adds param_Value_origin
+            if param_value_origin != '' and param_value_origin is not None:
+                new_dict[const.PARAM_VALUE_ORIGIN] = param_value_origin
+            else:
+                # puts it empty for user to fill it out mandatorily
+                new_dict[const.PARAM_VALUE_ORIGIN] = 'Added by Control Owner'
```

Comment:
This value needs to be changed after Prashant confims which is the default value for the field. @vikas-agarwal76 . In the meantime I'm using a placeholder value

Reply:
Is it required to populate the value for param value origin?